public/app/plugins/datasource/loki/LanguageProvider.test.ts

```diff
@@ -237,6 +237,15 @@ describe('Language completion provider', () => {
       expect(labelValues).toEqual(['label1_val1', 'label1_val2']);
     });
 
+    it('fetch label when options.streamSelector provided and values is not cached', async () => {
```

Comment:
what's this test testing? I notice it still passes if I comment all the new `streamSelector` and `streamParam` logic in `fetchLabelValues`

Reply:
It was testing that the request is successfully called and we receive correct values, but you are right, we weren't testing if `streamSelector` was actually included in request. I've updated both tests to correctly check if streamSelector was added. Thanks for catching this!